src/admin-views/aggregator/origins/refine.php

```diff
@@ -85,12 +73,13 @@ class="tribe-ea-field tribe-ea-size-xlarge"
 			>
 			<span class="tribe-bumpdown-trigger tribe-bumpdown-permanent tribe-bumpdown-nohover tribe-ea-help dashicons dashicons-editor-help" data-bumpdown="<?php echo esc_attr( $radius->help ); ?>" data-width-rule="all-triggers"></span>
 		</div>
+		<?php if ( 'eventbrite' !== $origin_slug ) : ?>
 		<div class="tribe-refine">
 			<?php
-				$start = empty( $record->meta['start'] ) ? '' : $record->meta['start'];
-				if ( is_numeric( $start ) ) {
-					$start = date( Tribe__Date_Utils::DATEONLYFORMAT, $start );
-				}
+			$start = empty( $record->meta['start'] ) ? '' : $record->meta['start'];
+			if ( is_numeric( $start ) ) {
+				$start = date( Tribe__Date_Utils::DATEONLYFORMAT, $start );
```

Comment:
:no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

<sub>hash: 2f308f0bf1c220a90f965452622519d5</sub>